docs/api-testing/sauce-connect.md

```diff
@@ -21,15 +21,40 @@ Sauce Labs API Testing fully integrates with Sauce Connect Proxy tunnels, enabli
 
 ## Start API-Specific Tunnel
 
-1. First, you'll need to create a .yaml configuration file. Copy and paste the sample template below into a text editor file and save it as a .yaml file. In this example, we'll name it **api-config.yaml**.
-     ```yaml title="Sample Template (api-config.yaml)"
-     ---
-     rest-url: "https://api.us-west-4-i3er.saucelabs.com/rest/v1"
-     user: "SAUCE_USERNAME"
-     api-key: "SAUCE_ACCESS_KEY"
-     vm-version: "v2alpha"
-     tunnel-identifier: "SAUCE_USERNAME_TUNNEL"
-     ```
+1. First, you'll need to create a .yaml configuration file. Copy and paste the sample template below into a text editor and save it as a .yaml file. In this example, we'll name it **api-config.yaml**.
+  <Tabs
+      defaultValue="US Data Center"
+      values={[
+        {label: 'US Data Center', value: 'US Data Center'},
+        {label: 'EU Data Center', value: 'EU Data Center'},
+      ]}>
+
+  <TabItem value="US Data Center">
+
+  ```yaml
+  ---
+  rest-url: "https://api.us-west-4-i3er.saucelabs.com/rest/v1"
+  user: "$SAUCE_USERNAME"
+  api-key: "$SAUCE_ACCESS_KEY"
+  vm-version: "v2alpha"
+  tunnel-identifier: "$SAUCE_USERNAME_TUNNEL"
+  ```
+
+  </TabItem>
+  <TabItem value="EU Data Center">
+
+  ```yaml
+  ---
+  rest-url: "https://api.eu-west-3-lnbf.saucelabs.com/rest/v1"
+  user: "$SAUCE_USERNAME"
+  api-key: "$SAUCE_ACCESS_KEY"
+  vm-version: "v2alpha"
+  tunnel-identifier: "$SAUCE_USERNAME_TUNNEL"
+  ```
```

Comment:
Hi @LoryApiFortress @waggledans , would you pls take a look for accuracy? 

Also, I wanted to double-check on two  yaml keys that I noticed were removed from the original version -- these are no longer being used?

```yaml
---
no-remove-colliding-tunnels: true
tunnel-cert: private
```

Reply:
@kimsauce as far as I know the two keys are still valid and used

Reply:
Hi @waggledans @alexh-sauce , can you confirm if `no-remove-colliding-tunnels` and `tunnel-cert` are still valid? In the CLI doc, `no-remove-colliding-tunnels` says "deprecated" and `tunnel-cert` is missing. They both appear to be working for API Testing tunnels (just ran a test to confirm).

Reply:
Yes, please remove these commands.

`--tunnel-cert private` is for the private certificate that is not recommended for customer use. The default is a publicly signed certificate.

`--no-remove-colliding-tunnels` is deprecated, and replaced with `--tunnel-pool`. Using this flag is part of the High Availability Setup, which is documented separately.